package.json

```diff
@@ -11,12 +11,12 @@
   },
   "scripts": {
     "dev": "concurrently \"npm run dev:client\" \"npm run dev:lambda\"",
-    "dev:client": "preact watch",
+    "dev:client": "preact watch --refresh false",
     "dev:lambda": "netlify-lambda serve src/lambda",
     "start": "npm run -s server",
     "prestart": "npm run -s build",
     "server": "superstatic build -p ${PORT:-8080} --host 0.0.0.0 --gzip -c '{\"rewrites\": [{\"source\":\"**\",\"destination\":\"index.html\"}],\"headers\":[{\"source\":\"**\",\"headers\":[{\"key\":\"Cache-Control\",\"value\":\"max-age=31536000\"}]}]}'",
-    "build": "netlify-lambda build src/lambda && preact build --prerenderUrls src/prerender.js --inline-css false",
```

Comment:
Note to self: Add back in when `next.3` is published, as Critters was temporarily removed in `next.2` due to not being compatible with newer `html-webpack-plugin` versions in different package managers.